src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/RatingControlTests/RatingControlPage.xaml.cs

```diff
@@ -0,0 +1,639 @@
+				rifi.Glyph = "\uEB52";
+				rifi.UnsetGlyph = "\uEB52";
+				rifi.PointerOverGlyph = "\uEB52";
+				rifi.PointerOverPlaceholderGlyph = "\uEB52";
+				rifi.DisabledGlyph = "\uEB52";
+
+				CustomImagesTwo.ItemInfo = rifi;
+
+				DependencyObject obj = FindVisualChildByName(CustomImagesTwo, "RatingBackgroundStackPanel");
+				var child = VisualTreeHelper.GetChild(obj, 0);
+				AutomationProperties.SetAutomationId(child, "CustomImagesTwo_FirstTextItem");
+				AutomationProperties.SetAccessibilityView(child, Windows.UI.Xaml.Automation.Peers.AccessibilityView.Control);
+
+				CustomImagesTwoLoadedStageTwoCheckBox.IsChecked = true;
+			}
+			else
+			{
+				var imageInfo = new RatingItemImageInfo();
+				imageInfo.Image = new Windows.UI.Xaml.Media.Imaging.BitmapImage(new Uri("ms-appx:/Assets/rating_set.png"));
+				imageInfo.UnsetImage = new Windows.UI.Xaml.Media.Imaging.BitmapImage(new Uri("ms-appx:/Assets/rating_unset.png")); ;
+				imageInfo.PlaceholderImage = new Windows.UI.Xaml.Media.Imaging.BitmapImage(new Uri("ms-appx:/Assets/rating_placeholder.png")); ;
+				imageInfo.DisabledImage = new Windows.UI.Xaml.Media.Imaging.BitmapImage(new Uri("ms-appx:/Assets/rating_disabled.png")); ;
+				imageInfo.PointerOverImage = new Windows.UI.Xaml.Media.Imaging.BitmapImage(new Uri("ms-appx:/Assets/rating_mouseoverset.png")); ;
+				imageInfo.PointerOverPlaceholderImage = new Windows.UI.Xaml.Media.Imaging.BitmapImage(new Uri("ms-appx:/Assets/rating_mouseoverplaceholder.png"));
+
+				CustomImagesTwo.ItemInfo = imageInfo;
+
+				DependencyObject obj = FindVisualChildByName(CustomImagesTwo, "RatingBackgroundStackPanel");
+				var child = VisualTreeHelper.GetChild(obj, 0);
+				AutomationProperties.SetAutomationId(child, "CustomImagesTwo_FirstImageItem_Again");
+				AutomationProperties.SetAccessibilityView(child, Windows.UI.Xaml.Automation.Peers.AccessibilityView.Control);
+
+				CustomImagesTwoLoadedStageThreeCheckBox.IsChecked = true;
+			}
+		}
+
+		private void CustomImagesTwo_Loaded(object sender, RoutedEventArgs e)
+		{
+			DependencyObject obj = FindVisualChildByName(CustomImagesTwo, "RatingBackgroundStackPanel");
+			var child = VisualTreeHelper.GetChild(obj, 0);
+			AutomationProperties.SetAutomationId(child, "CustomImagesTwo_FirstImageItem");
+			AutomationProperties.SetAccessibilityView(child, Windows.UI.Xaml.Automation.Peers.AccessibilityView.Control);
+
+			CustomImagesTwoLoadedStageOneCheckBox.IsChecked = true;
+		}
+
+		private void TestRatingControl_Loaded(object sender, RoutedEventArgs e)
+		{
+			_dt = new DispatcherTimer();
+			_dt.Interval = TimeSpan.FromSeconds(0.1);
+			_dt.Start();
+			_dt.Tick += DispatcherTimer_Tick;
+
+			DependencyObject obj = FindVisualChildByName(TestRatingControl, "RatingBackgroundStackPanel");
+			UIElement ui = obj as UIElement;
+			DependencyObject child = VisualTreeHelper.GetChild(obj, 1);
+			TextBlock tb = child as TextBlock;
+			this._secondTextBlockUI = tb;
+
+			//SetupAnimatedValuesSpy();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6162540)